web/static/js/components/simulation/QuestionnaireSimulation.jsx

```diff
@@ -33,18 +34,27 @@ type Props = {
   projectId: number,
   questionnaireId: number,
   mode: string,
-  questionnaireActions: Object
+  questionnaireActions: Object,
+  t: Function
 }
 
 type State = {
   simulation: ?Simulation
 }
 
-class QuestionnaireSimulation extends Component<Props, State> {
+const QuestionnaireSimulation = translate()(class extends Component<Props, State> {
```

Comment:
connect with translate at the bottom of the file with the export default.

Reply:
Done!